### ISSUE TITLE
Strategy for Unblocking the Review of Out-of-Date PRs

### ISSUE THREAD

**Issue by epiro-ps** [ArdanaLabs/dUSD#344]
Due to our varying timezones and hours, we often get into the situation where a PR that could be reviewed and potentially merged is rendered invalid because another PR was merged into master making that branch out of date and potentially regressive. This can cost us a day cycle because it relies on the author becoming aware of the issue and performing an update to the branch. A solution I'm proposing here to remedy this would be to follow this decision tree:

- If a PR is out of date with master and there are no conflicts, any developer can update the branch with master.
  - If the update doesn't pass all the test suites and bugs are found, assign the PR back to the Author.
  - Else conduct a review.
- Else assign the PR back to the Author.
